catalog/task/push-sbom-to-pyxis/0.1/push-sbom-to-pyxis.yaml

```diff
@@ -0,0 +1,41 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: push-sbom-to-pyxis
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.12.1"
+    tekton.dev/tags: release
+spec:
+  description: >-
+    Tekton task that extracts sbom from a pull spec and pushes it to Pyxis.
+  params:
+    - name: imageURL
+      description: URL of the main binary image.
+      type: string
+  volumes:
+    - name: workdir
+      emptyDir: {}
+  steps:
+    - name: download-sbom-for-image
+      # Image taken from https://github.com/sigstore/cosign/pkgs/container/cosign%2Fcosign
+      image:
+        ghcr.io/sigstore/cosign/cosign:03468a1498876ccfb72e06f63fdb4f1c562dbca1
+      args:
+        - download
+        - sbom
+        - --output-file
+        - /workdir/sbom-cyclonedx.json
+        - $(params.imageURL)
+      volumeMounts:
+        - mountPath: /workdir
+          name: workdir
+    - name: check-file
+      image: quay.io/hacbs-release/release-utils:latest
+      script: |
+        ls -l /workdir
```

Comment:
What is this for? It will always work and it doesn't provide a good output (the more task runs, the more noise).

Reply:
This whole step can be considered temporary just to have something until the next story will implement the step to transform the data. You could say it's for testing. In which case it should be removed, I know. But again, it will be done properly in the next PR. I can change it to a simple "non-empty file exists" test or remove the step completely if you like.

Reply:
If it doesn't contribute anything to this PR, let's remove it.

Reply:
OK, removed, thanks.